guides/common/modules/proc_adding-a-host-to-a-host-group.adoc

```diff
@@ -3,6 +3,8 @@
 
 You can add a host to a host group in the {ProjectWebUI}.
 
+NOTE: Adding a host to a host group updates the host settings to the values defined in the host group.
```

Comment:
Is there a reason for not using the Asciidoc note format?

Reply:
I think so far we have somestimes used "NOTE: ..." if it was only a single line. I am leaning towards `==== > [NOTE] > .... > ====`, but am OK with both.

I would be happy to create a PR to fix this if we agree on it beforehand.

Reply:
Per [Asciidoctor Docs](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), a single para note is to be introduced by `NOTE:`. Using `[NOTE]` with a block is to be used for complex content (more than a single paragraph, I assume).

Based on this, I'd prefer not to mandate block notes only. Or is there a reason to avoid simple single para notes that I'm not seeing? @apinnick

Reply:
The [RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions) uses the block style for admonitions. [OpenShift docs](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions) requires the block style. I do not know whether RHEL has a similar requirement.

Reply:
@apinnick I asked around and RHEL apparently has the same guideline: use code blocks rather than the "shorthand" version. I'll fix it here and let @maximiliankolb decide if he wants to fix it repo-wide :)

Reply:
Or I could do it with yet another huge 'sed' command since I have so much time on my hands. :-)

Reply:
Surprisingly, there are only 21 occurrences of `NOTE: text` in the repo! That's doable.